lib/mozart_fetcher/fetcher.ex

```diff
@@ -12,6 +12,7 @@ defmodule MozartFetcher.Fetcher do
   def process(components) do
     ExMetrics.timeframe "function.timing.fetcher.process" do
       components
+      |> Enum.with_index
       |> Enum.map(&Task.async(fn -> Component.fetch(&1) end))
```

Comment:
I thought we might need to specify the index here, but it looks like the integration tests work so it looks like we wouldn't need to.